docs: clarify the parallelCodeSplitting and removeAvailableModules options (#10368)
web-infra-dev/rspack: website/docs/en/config/optimization.mdx

};
```

:::danger

> It is danger to disable this optimization, as doing so may not only significantly increase the bundle size but also greatly slow down the build process.

Reviewer: ```suggestion
Disabling this optimization is dangerous, as it may significantly increase the bundle size and greatly slow down the build process.
```
